src/Weaviate.Client/Rest/Users.cs

```diff
@@ -65,33 +73,57 @@ internal partial class WeaviateRestClient
             ?? throw new WeaviateRestClientException();
     }
 
-    internal async Task<bool> UserDbDelete(string userId)
+    internal async Task UserDbDelete(string userId)
```

Comment:
Sorry if there was a missunderstanding, but the bool here should stay. 

I think the distinction for me is:
- if I want to delete something, it is gone afterwards. So getting an exception if it is not there already feels wrong
- there might be some cases, where we DO want to be sure that it actually deleted something. Then the bool is helpful